requirements.txt

```diff
@@ -1,4 +1,4 @@
-git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
+XBlock==0.4.14
 git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
```

Comment:
Maybe it is worthwhile to add a comment that since [PyPi](https://pypi.python.org/pypi/xblock-openedx-utils) doesn't have the latest version of xblock-utils, this is to install xblock-utils from git.